charts/tractusx-connector-azure-vault/subcharts/omejdn/README.md

```diff
@@ -0,0 +1,39 @@
+# daps
+
+![Version: 0.0.1](https://img.shields.io/badge/Version-0.0.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.0.1](https://img.shields.io/badge/AppVersion-0.0.1-informational?style=flat-square)
```

Comment:
pls rebase onto latest `main` - the omejdn sub charts are gone since `0.5.0-rc1`